### PR TITLE
[TR] Add Installation/macOS

### PR DESCRIPTION
**Pull request by zeusminus**
Add Turkish translations for `Installation/macOS` article

### CODE REVIEW

File: wiki/Installation/macOS/tr.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- macOS çalıştıran ortalama bir bilgisayar.
+
+## osu!'yu kurmak
+
+osu!'nun macOS sürümü deneyseldir. Oyun işlevsel bir şekilde çalışmayabilir, hatta açılmayabilir. Bu [eğitici video](https://www.youtube.com/watch?v=3ay221bMHNo) sizin aşağıdaki süreci anlamanıza yardımcı olacaktır.
+
+1. Resmi olmayan [indirme sayfasına](https://osu.ppy.sh/community/forums/topics/682197) gidin, `osu!.app.zip` dosyasını indirin ve dizine çıkarın.
+2. Bir uygulama gibi gözüken `osu!.app` klasörünü göreceksiniz. Üzerine sağ tıklayın, ve `Aç`'a tıklayın.
+   - macOS uygulamanın tanımlanamayan bir geliştirici tarafından geliştirilmiş olarak tanıyabileceğinden, dosyaya çift sol tıklamanız onu açmayabilir. Sağ tıklamak ve `Aç`'ı seçmek bu sorunu çözecektir.
+3. Yükleyici otomatik olarak çalışacaktır. Kurulum konumu değiştirilemez ve doğrudan `osu!.app` üzerine kurulur. `osu!.app`'i taşımanız onunla birlikte gelen tüm dosyaları da beraberinde taşıyacaktır. Klasör içerisindeki dosyaları görüntülemek için sağ tıklayıp `Paket İçeriğini Göster` seçeneğini de seçebilirsiniz.
```
Comment (goeo-) [inline wiki/Installation/macOS/tr.md:16]: would be best if we checked the official macOS translation of `Paket İçeriğini Göster`

File: wiki/Installation/macOS/tr.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+4. Yükleyici tamamlandığında osu! açılacaktır. osu!direct kullanılarak birtakım başlangıç beatmapleri sizin için indirilecektir.
+5. osu!, giriş yapmanız veya kayıt olmanız için istemde bulunacaktır.
```
Comment (goeo-) [inline wiki/Installation/macOS/tr.md:18]: not natural - `isteyecektir` instead? lol

File: wiki/Installation/macOS/tr.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Bu sayfa size macOS cihazınızda osu!'yu nasıl çalıştıracağınızı kabaca açıklayacaktır.
+
+## Minimum sistem gereksinimleri
+
+- macOS çalıştıran ortalama bir bilgisayar.
+
+## osu!'yu kurmak
+
+osu!'nun macOS sürümü deneyseldir. Oyun işlevsel bir şekilde çalışmayabilir, hatta açılmayabilir. Bu [eğitici video](https://www.youtube.com/watch?v=3ay221bMHNo) sizin aşağıdaki süreci anlamanıza yardımcı olacaktır.
```
Comment (goeo-) [inline wiki/Installation/macOS/tr.md:11]: maybe specify that the video is in english

File: wiki/Installation/macOS/tr.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# macOS için kurulum
+
+Bu sayfa size macOS cihazınızda osu!'yu nasıl çalıştıracağınızı kabaca açıklayacaktır.
+
+## Minimum sistem gereksinimleri
+
+- macOS çalıştıran ortalama bir bilgisayar.
```
Comment (goeo-) [inline wiki/Installation/macOS/tr.md:7]: `yeterince iyi`? `düzgün`? - `decent` probably shouldn't be translated as `ortalama` 

funny how osu wiki be like that though

Comment (zeusminus) [reply wiki/Installation/macOS/tr.md:7]: meh, "decent" here can perfectly be translated as "ortalama" here. another possibility would be translating it as "iyi" which I used in the original Installation article

**Pull request merged**